impeller/typographer/backends/skia/text_frame_skia.cc

```diff
@@ -57,42 +59,49 @@ std::shared_ptr<TextFrame> MakeTextFrameFromTextBlobSkia(
   for (SkTextBlobRunIterator run(blob.get()); !run.done(); run.next()) {
     // TODO(jonahwilliams): ask Skia for a public API to look this up.
     // https://github.com/flutter/flutter/issues/112005
+
     SkStrikeSpec strikeSpec = SkStrikeSpec::MakeWithNoDevice(run.font());
     SkBulkGlyphMetricsAndPaths paths{strikeSpec};
+    AxisAlignment alignment = AxisAlignment::kNone;
+    if (run.font().isSubpixel()) {
+      switch (
+          strikeSpec.createScalerContext()->computeAxisAlignmentForHText()) {
+        case SkAxisAlignment::kNone:
+          // Skia calls this case none, meaning alignment in both X and Y.
```

Comment:
Good call and thanks for documenting. Could also move it into a method `AxisAlignment ToAxisAligment(SkAxixalignment align);` with the requisite documentation.

Reply:
Done